config/application.rb

```diff
@@ -1,6 +1,19 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require "rails"
+%w(
+  active_record
+  action_controller
+  action_view
+  action_mailer
+  active_job
+  sprockets
```

Comment:
Is there a way we can ask for this list, as opposed to listing it ourselves?  If possible would prefer something like `Rails.frameworks - ['test_unti'].each do |framework|` or something like that.